README.md

```diff
@@ -184,15 +210,37 @@ jlpm run build
 jupyter lab build
 ```
 
-Start JupyterLab in watch mode:
+You can now start JupyterLab and check if your extension is work fine:
+
+```bash
+jupyter lab
+```
+
+### Change the Sources
+
+If you want to develop and iterate on the code, you will need to open 2 terminals.
+
+In terminal 1, go to the extension folder and run the following:
+
+```bash
+jlpm watch
+```
+
+Then in terminal 2, start JupyterLab with the watch flag:
 
 ```bash
 jupyter lab --watch
 ```
 
+From there, you can change your extension source code, it will be recompiled,
+and you can refresh your browser to see your changes.
+
+We are using [embedme](https://github.com/zakhenry/embedme) to embed code snippets into the markdown READMEs. If you make changes to the source code, ensure you update the README and run `yarn embedme` from the root of the repository to regenerate the READMEs.
```

Comment:
Should we mention `jlpm` instead of `yarn`?

Reply:
Done. Thx